src/form-fields/text-field.js

```diff
@@ -56,6 +56,7 @@ class TextFieldComponent extends React.Component {
                     {...other}
                     value={this.state.value}
                     onChange={this.onChange}
+                    error={true}
```

Comment:
This change is hard to follow. Could this introduce regressions? I appears as if this would be applied to all text fields, but I can't see it being applied.

Regardless of that, you can just do `error` instead of `error={true}`

Reply:
😱 Yikes. No; that change is not supposed to be there. I was beginning to try to look at why validation errors weren't showing up, but I guess I made this debugging change before switching to a new branch.

Removed now. Thanks!